nni/retiarii/strategy/_rl_impl.py

```diff
@@ -7,7 +7,7 @@
 import gym
 import numpy as np
 import torch
-import torch.nn as nn
+# import torch.nn as nn
```

Comment:
What's wrong with this?
If you decided to delete this code, you'd better delete it instead of commenting it.

Reply:
okay, sorry, it's be good if not double be imported because torch is already was imported ```import torch```,for pros its less to maintenance
but i will delete the comment

Reply:
I think `import torch.nn as nn` if recommended as the official coding style. https://pytorch.org/tutorials/recipes/recipes/defining_a_neural_network.html